lib/compression/index.js

```diff
@@ -9,6 +9,7 @@
 /* eslint-env node */
 
 module.exports = {
+  zopfliWebpackPlugin: require('./gfx-webpack-plugin'),
```

Comment:
`gfx` is the github handle for the wasm binding lib. This file should probably be called zopfli-webpack-plugin